integration_test/gce/gce_testing.go

```diff
@@ -1302,8 +1302,12 @@ func attemptCreateInstance(ctx context.Context, logger *log.Logger, options VMOp
 	return vm, nil
 }
 
-func IsSUSE(imageSpec string) bool {
-	return strings.HasPrefix(imageSpec, "suse-") || strings.HasPrefix(imageSpec, "opensuse-") || strings.Contains(imageSpec, "sles-")
+func IsSLES(vm_OS_ID string) bool {
+	return vm_OS_ID == "sles" || vm_OS_ID == "sles_sap"
+}
+
+func IsSUSE(vm_OS_ID string) bool {
+	return vm_OS_ID == "opensuse" || vm_OS_ID == "opensuse-leap" || vm_OS_ID == "sles" || vm_OS_ID == "sles_sap"
 }
 
 func IsCentOS(imageSpec string) bool {
```

Comment:
Should we plan to change the rest of these functions in a subsequent PR? Or fold it into this one?

If we change them all to operate on the `vm` instead of `vm.ImageSpec`, we can clean them up one-by-one later…

Reply:
**> Should we plan to change the rest of these functions in a subsequent PR?**
I think we should update the rest of the functions, but i do expect unforseen consequences (other failures), so i propose to do the other updates in a followup PR.